docs/components/Try.md

```diff
@@ -104,20 +105,21 @@ export default {}
 </template>
 
 <script>
-import Vue from 'vue'
+import Vue from 'vue';
 Vue.component('ThisWillThrow', {
+  /* eslint-disable-next-line vue/require-render-return */
```

Comment:
Went back and forth on this. Unfortunately the only options here were to either put the comment directly in this one place or turn it off for all markdown files globally. And outside of this one edge case where we intentionally want a render function to throw, in every other case we'd want to be alerted if we wrote a bad render function. So putting the disable-next-line seems the lesser of the bad choices.

Reply:
Yeah, this makes sense. Another option might be to put /* eslint-disable vue/require-render-return */ to the top of the script section if we wanted to avoid extra noise right in the docs. But I dont feel strongly about it. What do you think?

Reply:
I think it being direct (I am only related to THIS line) is the clearest, and anyone familiar with ESLint will know to just ignore the comment. may be confusing (if placed anywhere visible) to someone unfamiliar with ESLint.

Reply:
Fair enough. Im fine with this.